package.json

```diff
@@ -114,7 +114,8 @@
     "sass-loader": "^10.0.5",
     "semver": "7.3.2",
     "source-map-explorer": "^2.5.3",
-    "ssv-keys": "https://github.com/bloxapp/ssv-keys.git",
+    "ssv-keys": "https://github.com/bloxapp/ssv-keys.git#1.0.7",
```

Comment:
Update version to 1.0.9 with new functionality for keeping the list of keyshare items and errors inside of them

Reply:
It's still in progress will deliver it with next PR

Reply:
in progress on our side, I have to check it again